packages/frontend-2/components/viewer/anchored-point/Thread.vue

```diff
@@ -162,7 +162,7 @@
               </FormButton>
             </div>
             <div
-              v-if="!canReply && !isEmbedEnabled"
+              v-if="!canReply && !isEmbedEnabled && !isLoggedIn"
```

Comment:
This bit i'm not sure about - what if the user is logged in, and yet still can't reply due to not having the rights to do so? Do we still show something to him?

Reply:
This will hide the "+" in the viewer for these users. So they are logged in, but can't comment, they won't see the plus.